wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Linker.kt

```diff
@@ -415,45 +416,44 @@ class Linker {
     return Linker(this, context)
   }
 
-  fun addError(format: String, vararg args: Any) {
+  fun addError(message: String) {
```

Comment:
Breaking API change

Reply:
Alternative: call into an expect/actual `format(String, vararg Any)` that throws on JS and Native.

Reply:
That would mean you couldn't use the Linker and thus couldn't actually load protos on those platforms which was the whole purpose of moving this artifact in the first place.

I can leave a `@Deprecated(level=ERROR)` function that only works on the JVM for binary compatibility if we want to maintain compatibility.

Reply:
Yeah, I meant you can leave the method with the old signature and make it call into the new one. That would obviously fail if somebody calls the old method on unsupported platforms.

Reply:
I think we take breaking changes here.